.github/workflows/build-package-docs.yaml

```diff
@@ -55,6 +55,9 @@ jobs:
   build-package-docs:
     runs-on: ubuntu-latest
     steps:
+    - name: Display workflow input parameters
+      run: echo "${{ toJSON(github.event.inputs) }}"
+      
```

Comment:
```suggestion

```

Reply:
Hmm. I'm not sure if this suggestion is going to do what I want. There should be an empty line between each step in the workflow. In this case that empty line has several space characters that shouldn't be there.